crates/chainio/clients/avsregistry/src/writer.rs

```diff
@@ -1022,6 +1239,81 @@ mod tests {
         assert!(tx_status);
     }
 
+    #[tokio::test]
+    async fn test_set_add_strategies() {
+        let (_container, http_endpoint, _ws_endpoint) = start_m2_anvil_container().await;
+
+        let private_key = FIRST_PRIVATE_KEY.to_string();
+
+        let avs_writer = build_avs_registry_chain_writer(http_endpoint.clone(), private_key).await;
+
+        let quorum_number = 0;
+        let strategy_params = [StrategyParams {
+            strategy: address!("54945180dB7943c0ed0FEE7EdaB2Bd24620256bc"),
+            multiplier: U96::from(1),
+        }];
+
+        let tx_hash = avs_writer
+            .add_strategies(quorum_number, strategy_params.to_vec())
+            .await
+            .unwrap();
+        let tx_status = wait_transaction(&http_endpoint, tx_hash)
+            .await
+            .unwrap()
+            .status();
+
+        assert!(tx_status);
+
+        let contract_stake_registry =
+            StakeRegistry::new(avs_writer.stake_registry_addr, get_provider(&http_endpoint));
+        assert_eq!(
+            strategy_params[0].strategy,
+            contract_stake_registry
+                .strategyParams(quorum_number, "1".parse().unwrap())
+                .call()
+                .await
+                .unwrap()
+                .strategy
+        );
```

Comment:
nit. Let's extract the second part into a variable, for ease of reading:

```suggestion
        let expected_strategy = contract_stake_registry
            .strategyParams(quorum_number, "1".parse().unwrap())
            .call()
            .await
            .unwrap()
            .strategy;

        assert_eq!(strategy_params[0].strategy, expected_strategy);
```